src/main/java/bg/libapp/libraryapp/model/entity/Rent.java

```diff
@@ -0,0 +1,91 @@
+package bg.libapp.libraryapp.model.entity;
+
+import jakarta.persistence.*;
+
```

Comment:
Check this IntelliJ setting for disabling wildcard imports - https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports